pkgs/applications/networking/instant-messengers/tangram/default.nix

```diff
@@ -70,7 +70,7 @@ stdenv.mkDerivation rec {
 
   postPatch = ''
     substituteInPlace src/meson.build --replace "/app/bin/blueprint-compiler" "blueprint-compiler"
-    substituteInPlace {src/,}re.sonny.Tangram troll/gjspack/bin/gjspack \
+    substituteInPlace src/bin.js troll/gjspack/bin/gjspack \
```

Comment:
Is it not necessary to patch this file?
https://github.com/sonnyp/Tangram/blob/eb563e9147df9eb759ec7b1355c3286fa7c3fd87/src/local.js#L1

Reply:
Yep, I believe this file is for development only https://github.com/sonnyp/Tangram/blob/eb563e9147df9eb759ec7b1355c3286fa7c3fd87/src/local.js#L20